### DISCUSSION TITLE
Manual preloading with v6.2?

### DISCUSSION THREAD

**Discussion by chester108** [jspsych/jsPsych#2295] [Q&A] [answered]
Hello :) I have an experiment created with v6.2 which requires manual media preloading. I noticed the preload plug-in is not included in the 6.2 library. Is manual preloading possible with this jsPsych version or only with newer releases? Thanks!

**Comment by jodeleeuw** [comment, marked as answer]
Hi @chester108 

You can preload in 6.2 using the `preload_images` argument in `jsPsych.init()`

```js
jsPsych.init({
	preload_images: ['img1.png', 'img2.png']
})
```

**Reply by chester108** [reply]
Hi @jodeleeuw thanks so much for the reply! :)
